core/modules/contact/contact.admin.inc

```diff
@@ -5,6 +5,23 @@
  * Admin page callbacks for the Contact module.
  */
 
+/**
+ * Page callback: Contact settings.
+ *
+ * @see contact_menu()
+ */
+function contact_settings_form($form, &$form_state) {
+  $form['#config'] = 'contact.settings';
+  $form['copy'] = array(
+    '#type' => 'checkbox',
+    '#title' => t("Show the 'Send yourself a copy' checkbox"),
+    '#description' => t("Allow users to send themselves a copy on site-wide and personnal contact form"),
```

Comment:
The word "personal" only has one "n".
I think it is not necessary to have the "on site-wide and personnal contact form" part, as that is what the setting is for.
However, I think adding something like "of their message" would not be a bad idea.
Also, it needs a period at the end.
So, something like this: "Allow users to send themselves a copy of their message."